src/main/java/no/nav/familie/integrasjoner/config/ApplicationConfig.kt

```diff
@@ -55,6 +58,21 @@ class ApplicationConfig {
                 .build()
     }
 
+    @Bean("propagateAuth")
+    fun restTemplatePropagateAuth(inRequest: HttpServletRequest,
+                                  consumerIdClientInterceptor: ConsumerIdClientInterceptor): RestOperations {
+
+        return RestTemplateBuilder()
+            .interceptors(ClientHttpRequestInterceptor { outRequest, body, requestExecution ->
```

Comment:
Hvorfor kan man ikke gjøre det som i onbehalfof-ruta i ba-sak?
@bent-lorentzen Kan du titte på denne?

Reply:
on-behalf fungerer vel bare for azureAd. Hvis Saf har lagt til støtte for azureAd, så kan vi nok bruke det, hvis ikke så er vel dette det nærmeste man kommer on-behalf med sts?

Reply:
@bent-lorentzen Det var snakk om å bruke azureAd. Saf skal ha fått støtte for Azuere V2, men @tnarland er mannen som kan bekrefte/avkrefte

Reply:
@stigebil Det stemmer. Saf støtter Azure V2 nå. Grunnen til vi ikke kan gjøre det likt som i ba-sak er nettopp opp fordi det er allerede satt opp der. Jeg har testet med chained onbehalfof-onbehalfof konfig også, men det har ikke funka..

Reply:
Det burde jo fungere med chained onbehalfof-onbehalfof konfig. Det brukes jo i dag når ba kaller ba-sak kaller integarsjoner sin tilgangskontroller som igjen henter nytt obo-token for å kalle azureAd og sjekke rettigheter.
Hvordan var det konfigurasjonen din med obo-autorisasjon i integrasjoner så ut?

Reply:
Jeg har ikke testet med konfigurasjon satt opp direkte i integrasjoner, bare manuelt via postman. Men uansett så forutsetter det vel at også SAF-applikasjonen er satt opp til å støtte obo-flyt?
Han jeg snakket med fra SAF-teamet visste ikke om de gjorde det, så jeg vet ikke om det vil fungere uten å bestille flere config-endringer fra de evt..
Det de har gjort er å legge til familie-integrasjoner blant deres "accepted audience", slik at det funker selv om scope bare er satt til familie-integrasjoner, som er grunnen til at man kan bruke samme obo-token inn og ut mot SAF...

Reply:
Mot azureAd fungerer det fint å lage chained obo-token manuelt via Postmann også, så da er det nok bare SAF som ikke støtter den flyten. Tenker derfor det får bli sånn som det er satt opp nå, enn så lenge ihvertfall..